tests/constants/admincli/cmd.go

```diff
@@ -82,4 +82,13 @@ const (
 
 	// ReadWriteAccess read-write rights for the volume
 	ReadWriteAccess = "read-write"
+
+	// ListVmgroupVMs referring to vmdkops_admin vmgroups vm ls
+	ListVmgroupVMs = vmdkopsAdmin + "vmgroup vm ls --name="
+
+	// VMlist VM of the vmgroup
+	VMlist = " --vm-list="
+
+	// AddDatastoreToVMgroup adds datastore to vmgroup
+	AddDatastoreToVMgroup = vmdkopsAdmin + "vmgroup access add --name="
```

Comment:
`AddDatastoreToVMgroup ` sounds confusing .. I guess idea is to add datastore access to VMgroup. .. it can be simplified with name `AddDatastoreAccess` or `AssignDatastoreAccess`